source/_integrations/waqi.markdown

```diff
@@ -8,9 +8,9 @@ ha_release: 0.34
 ha_iot_class: Cloud Polling
 ---
 
-The `waqi` sensor platform will query [World Air Quality Index](http://aqicn.org) service to check AQI value for a specific set of locations. The resulting indexes will be added to the Home Assistant as sensor outputs.
+The `waqi` sensor platform will query [World Air Quality Index](https://aqicn.org/) service to check AQI value for a specific set of locations. The resulting indexes will be added to the Home Assistant as sensor outputs.
```

Comment:
This redirects back to the HTTP version. So while a valid link, it now causes an unneeded redirect.